core/templates/dev/head/pages/collection_editor/editor_tab/collection_node_creator_directive.html

```diff
@@ -5,8 +5,8 @@
     <button class="btn btn-success" type="submit" ng-disabled="!newExplorationTitle">Add New Exploration</button>
   </form>
   or
-  <form ng-submit="addExploration()">
-    <input ng-model="newExplorationId" type="text" placeholder="New exploration ID">
-    <button type="submit" ng-disabled="!newExplorationId">Add Existing Exploration</button>
+  <form class="form-inline" ng-submit="addExploration()">
+    <input type="text" ng-model="newExplorationId" placeholder="Search explorations by name" typeahead="exploration for exploration in fetchExplorationsMetadata($viewValue)" class="form-control">
```

Comment:
Due to the placeholder, the input box should probably be ~5px wider (or at least wide enough to show `Search explorations by name` fully (right now it's partially cut off).